Add thread to custom instrumentation (#2)
kaylareopelle/docs-website: src/content/docs/apm/agents/ruby-agent/api-guides/ruby-custom-instrumentation.mdx

end
```

For more information, see [Tracer#start_transaction_or_segment in the New Relic RubyDoc](https://www.rubydoc.info/gems/newrelic_rpm/NewRelic/Agent/Tracer#start_transaction_or_segment-class_method).

### Instrumenting Threads

> To support applications that want some threads instrumented by New Relic, a class is available in 8.7.0 and above, `NewRelic::TracedThread`, that will create a thread that includes New Relic instrumentation. 

Reviewer: ```suggestion
To support applications that want to control which threads are instrumented by New Relic, a class is available in 8.7.0 and above, `NewRelic::TracedThread`, that will create a thread that includes New Relic instrumentation. 
```

> Use this class as a replacement for the native Thread class. Please see our [API documentation](https://www.rubydoc.info/gems/newrelic_rpm/NewRelic) for more details. 

Reviewer: I wonder if we can hack this link in advance? I think it'll end up being /NewRelic/TracedThread

> Use this class as a replacement for the native Thread class. Please see our [API documentation](https://www.rubydoc.info/gems/newrelic_rpm/NewRelic) for more details. 

Reviewer: ```suggestion
Use this class as a replacement for the native Thread class. Please see our [API documentation](https://www.rubydoc.info/gems/newrelic_rpm/NewRelic/TracedThread) for more details. 
```

```
  NewRelic::TracedThread.new do 

>     # custome code to run inside of the thread

Reviewer: ```suggestion
    # custom code to run inside of the thread
```
